### ISSUE TITLE
Rules pattern a11y issues

### ISSUE THREAD

**Issue by jamesjacobs** [jadu/pulsar#1180]
### Issue 1
Axe complains about heading order in rules heading (currently `h5`).

### Issue 2
Unable to navigate with a screen reader to the form in the `tfoot`. Only tested on Mac VO at time of writing.

**Comment by Stanton** [comment]
**Issue 1**
Doesn't feel appropriate to replace these with H1 elements, the general rule block sections are wrapped in fieldsets with a hidden legend (which interestingly doesn't seem to be properly announced with select2s, other fields are fine) so perhaps replace the `h5` with regular old `span` elements...

**Issue 2**
I seem to be able to navigate to this without issue, is there something I'm missing?
